src/content/tutorials/observability/multi-tenancy/creating-grafana-organization/_index.md

```diff
@@ -1,74 +1,38 @@
 ---
 linkTitle: Creating a Grafana organization
 title: Creating a Grafana organization
-description: Guide explaining how to manage Grafana organizations in the Observability Platform.
+description: Step-by-step guide to create and configure Grafana organizations for multi-tenant observability.
 menu:
   principal:
     identifier: tutorials-observability-multitenancy-create-grafana-organization
     parent: tutorials-observability-multitenancy
 weight: 40
 last_review_date: 2025-06-11
 user_questions:
-  - How to create a grafana organization?
-  - How to access multi-tenant observability data?
+  - How to create a Grafana organization?
+  - How to configure RBAC for Grafana organizations?
+  - How to assign tenants to organizations?
 owner:
   - https://github.com/orgs/giantswarm/teams/team-atlas
 ---
 
-When you first log in to [your installation's `Grafana`]({{< relref "/tutorials/observability/data-exploration/accessing-grafana" >}}), you see the preselected `Grafana` organization called `_Shared Org_`. This shared organization contains a curated set of managed dashboards available to everyone with `Grafana` access.
+This guide walks you through creating and configuring Grafana organizations to implement [multi-tenancy]({{< relref "/tutorials/observability/multi-tenancy" >}}) in your observability platform.
```

Comment:
It is also worth mentioning that a GrafanaOrganization is what dictate the validity of a tenant.

Reply:
```suggestion
This guide walks you through creating and configuring Grafana organizations to implement [multi-tenancy]({{< relref "/tutorials/observability/multi-tenancy" >}}) in your observability platform. We strongly advise that you read this documentation first.
```

Reply:
:+1:

Reply:
:+1: